IDE/MPLabX/wolfBoot-SAME51.X/Makefile

```diff
@@ -0,0 +1,113 @@
+#
```

Comment:
Please consider using `IDE/MPLABX` to match our other libraries and Microchip naming.